test/integration/integration.test.js

```diff
@@ -54,7 +54,7 @@ const registerIntegrationTest = async (configPath) => {
 };
 
 // Increase test timeout to 10s (default 5s)
-jest.setTimeout(10000);
+jest.setTimeout(100000);
```

Comment:
The 100s seem to be a typo/leftover?

Reply:
Sorry for the delay. This was added because the tests run super slow on my system that would actually take more than a minute to finish. Of course, if you think the change should not be added to the repo, I can remove it.

Reply:
I just saw a 300sec commit to the master, I'm just going to merge that.